File: 조선빈/Lv.1/2016년.py
def WhatDay(b):
    if b%7 == 0:
        answer = 'THU'
    if b%7 == 1:
        answer = 'FRI'
    if b%7 == 2:
        answer = 'SAT'
    if b%7 == 3:
        answer = 'SUN'
    if b%7 == 4:
        answer = 'MON'
    if b%7 == 5:
        answer = 'TUE'
    if b%7 == 6:
        answer = 'WED'
    return answer
def solution(a, b):
    answer = ''
    
    Jan = list(range(1,32))
    Feb = list(range(1,30))
    Mar = Jan.copy()
    Apr = list(range(1,31))
    May = Jan.copy()
    Jun = Apr.copy()
    Jul = Jan.copy()
    Aug = Jan.copy()
    Sep = Apr.copy()
    Oct = Jan.copy()
    Nov = Apr.copy()
    Dec = Jan.copy()
    #Month = [Jan,Feb,Mar,Apr,May,Jun,Jul,Aug,Sep,Oct,Nov,Dec]
    
    if a == 1:
        date = []
    if a == 2:
        date = Jan
    if a == 3:
        date = Jan + Feb
    if a == 4:
        date = Jan + Feb + Mar
    if a == 5:
        date = Jan + Feb + Mar + Apr
    if a == 6:
        date = Jan + Feb + Mar + Apr + May
    if a == 7:
        date = Jan + Feb + Mar + Apr + May + Jun
    if a == 8:
        date = Jan + Feb + Mar + Apr + May + Jun + Jul
    if a == 9:
        date = Jan + Feb + Mar + Apr + May + Jun + Jul + Aug
    if a == 10:
        date = Jan + Feb + Mar + Apr + May + Jun + Jul + Aug + Sep
    if a == 11:
        date = Jan + Feb + Mar + Apr + May + Jun + Jul + Aug + Sep + Oct
    if a == 12:
        date = Jan + Feb + Mar + Apr + May + Jun + Jul + Aug + Sep + Oct + Nov
    answer = WhatDay(len(date)+b)
   
    return answer

File: 이유진/codes/Lv.1/2016년.java
class Solution {
    public String solution(int a, int b) {
        int date = 0;
        for(int i = 1; i < a; i++ ){
            if( i == 2) date += 29;
            if( (i == 1)||(i == 3)||(i == 5)||(i == 7)||(i == 8)||(i == 10)||(i == 12)) date += 31;
            if( (i == 4)||(i == 6)||(i == 9)||(i == 11)) date += 30;
        }
        date += b - 1;
        String answer = "";
        int day = date % 7;
        
        String[] arr = {"FRI","SAT","SUN","MON","TUE","WED","THU"};
        
        answer = arr[day];
        return answer;
    }
}